src/components/adaptive-cards/Spacer/Spacer.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import webexComponentClasses from '../../helpers';
+import {acPropTypes, registerComponent} from '../Component/Component';
+
+/**
+ * Spacer component
+ *
+ * @param {object} props  React props passed to the component
+ * @param {object} props.data  Component definition
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {object} [props.style]  Custom style to apply
+ * @returns {object} JSX of the component
+ */
+
+export default function Spacer({data, className, style}) {
+  const spacingType = data.spacing;
+  const isSeparator = data.separator;
+
+  const spacingClassName = `${data.childType}-spacing-${String(spacingType).toLowerCase()}`;
+  const separatorClassName = `${data.childType}-separator-${isSeparator}`;
```

Comment:
no need to suffix "isSeparator" (true), because the class is only added when true.